deptofdefense/move.mil
# Add Tutorials (models, controller, routes, views, seed data, fixtures, and specs)

## Checklist

I have…


## Summary of Changes

This pull request adds content (and related front-end assets) to the Tutorials (`/tutorials`) page. It also introduces two models, `Tutorial` and `TutorialStep`, as well as related migrations and a `tutorials_controller.rb`. Seed tutorial data (`db/seeds/tutorials.yml`) can be added to the database by running `bin/rails db:seed`.

On the testing side of things, the [factory_girl_rials gem](https://github.com/thoughtbot/factory_girl_rails) brings [Factory Girl](https://github.com/thoughtbot/factory_girl) fixtures to the test suite.

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
1. `git checkout add-tutorials`,
1. set up development dependencies according to `CONTRIBUTING.md`,
1. `bin/rails server`, and
1. load up http://localhost:3000/tutorials in your Web browser of choice.

## Screenshots

### Narrow Viewport

![narrow-fullpage](https://user-images.githubusercontent.com/27780860/28678887-8e299844-72bf-11e7-879e-6eabece1f1ba.png)

### Wide Viewport

![wide-fullpage](https://user-images.githubusercontent.com/27780860/28678884-8c0a66d8-72bf-11e7-920b-a0c504326f38.png)
